config/rbac/node-labeler_role.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: node-labeler
+rules:
+- apiGroups:
+  - security.openshift.io
+  resourceNames:
+  - privileged
```

Comment:
@mwalter-habana, why does a node labeler use the `privileged` SCC?

Reply:
I guess it's because it needs `hostPath` volume, right?

Reply:
It needs hostPath in order to write to the /etc/kubernetes/node-feature-discovery/features.d/ directory